README.md

```diff
@@ -1,2 +1,18 @@
 # warping-contracts
-Example Solidity contracts transpiled to Cairo using Warp
+
+Transpiles in Warp with no errors in Solidity 0.8.14 with:
+
+```shell
+bin/warp transpile exampleContracts/chainlink/pricefeed/pricefeedExample.sol
+```
+
+Modified:
+
+-put contract AggregatorV3Interface.sol 
```

Comment:
Did you try installing `chainlink` inside the warp folder using npm? I think that should add the dependencies you need so you don't have to use this get around to use the interface